internal/registry/registry.go

```diff
@@ -97,6 +110,15 @@ func scanPackages(packages map[string]*ast.Package) templateData {
 	return td
 }
 
+func lowerKind(kind string) (result string) {
```

Comment:
Not sure if we need the first case to be lower (e.g. `roleBinding `) but we'll definitely need it all lowercase for the resource names, and probably the registry too, because the URI will look like `/apis/core/v1alpha1/rolebindings`

Reply:
They should be all lowercase, imo.

Reply:
When @echlebek asked about snake case vs. camel case, I did not realize that we were talking about URLs. Sorry about the confusion.

Reply:
I'll change it to be lower-cased.